spdx/document.py

```diff
@@ -237,11 +252,14 @@ def validate(self, messages=None):
 
         return (self.validate_version(messages)
             and self.validate_data_lics(messages)
+            and self.validate_name(messages)
```

Comment:
we likely want the validation to be strict or not.... Note that this PR can still go through and this can be fixed afterwards

Reply:
I'm planning to handle the validation part after Phase 1 ? Is that okay ?

Reply:
yes